### PR TITLE
tests: simple search acceptance tests

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun samuele.kaplun@cern.ch

### CODE REVIEW

File: tests/acceptance/test_search.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    selenium.find_element_by_link_text('Authors').click()
+    assert 'Search 100 authors' in selenium.page_source
+    search_button = selenium.find_element_by_id('search-form-button').click()
+    assert 'Found 100 results.' in selenium.page_source
+
+
+@pytest.mark.xfail(reason='we need first to implement integration of data')
+def test_data(selenium):
+    selenium.find_element_by_link_text('Data').click()
+    assert 'Search 100 data sets' in selenium.page_source
```
Comment (jacquerie) [inline tests/acceptance/test_search.py]: @eamonnmag, can you point us to an endpoint we can use to get the number of records in HEPData?

**Comment by jacquerie** [comment]
These tests could be rewritten to use a spy (see: https://pypi.python.org/pypi/pytest-mock/1.2) on `render_template` to assert that it was called with the proper arguments. This would also mean that these tests would be moved to the integration suite, and merged with the existing ones. Or do we think it's important to test e2e these pages?

**Comment by jmartinm** [comment]
I am checking to try to reduce PRs open. Is this still relevant?

**Comment by kaplun** [comment]
I think it has its value, in the sense that it tries to see that the splash pages and regular searches are not crashing in any given PR.

**Comment by kaplun** [comment]
I shall rebase it.

**Comment by jacquerie** [comment]
Postponed, it will be resurrected if necessary.

**Pull request closed without merging**